pkg/util/mount/nsenter_mount.go

```diff
@@ -189,9 +189,10 @@ func (n *NsenterMounter) IsLikelyNotMountPoint(file string) (bool, error) {
 		return true, nil
 	}
 	strOut := strings.TrimSuffix(string(out), "\n")
-
 	glog.V(5).Infof("IsLikelyNotMountPoint findmnt output for path %s: %v", file, strOut)
-	if strOut == file {
+
+	//TODO: This is a temperory workaround for fixing the buggy code. Track the solution with issue #26996
```

Comment:
fixed

On Wed, Jun 8, 2016 at 11:20 AM, Saad Ali <notifications@github.com> wrote:

> In pkg/util/mount/nsenter_mount.go
> <https://github.com/kubernetes/kubernetes/pull/27054#discussion_r66310353>
> :
>
> >  	glog.V(5).Infof("IsLikelyNotMountPoint findmnt output for path %s: %v", file, strOut)
> > -	if strOut == file {
> > +
> > +	//TODO: This is a temperory workaround for fixing the buggy code. Track the solution with issue #26996
>
> nit: spelling "temporary"
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/27054/files/602e5aa032deae544e8560635ad3f48d30a7260d#r66310353>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ASSNxfRs7Rk_9i6mogZubOq-XCi7AdFlks5qJwfigaJpZM4IxMdY>
> .
>



-- 
- Jing